packages/gatsby-plugin-utils/src/polyfill-remote-file/__tests__/gatsby-image-resolver.ts

```diff
@@ -172,35 +169,28 @@ describe(`gatsbyImageData`, () => {
     const parsedSrcSet = parseSrcSet(result.images.sources[0].srcSet)
     expect(parsedSrcSet.length).toBe(4)
     expect(parsedSrcSet[0].src).toEqual(
-      `/_gatsby/image/${Buffer.from(portraitSource.url).toString(
-        `base64`
-      )}/${Buffer.from(`w=75&h=120&fm=avif&q=75`).toString(`base64`)}/${
-        portraitSource.basename
-      }.avif`
+      `/_gatsby/image/${base64URLEncode(portraitSource.url)}/${base64URLEncode(
+        `w=75&h=120&fm=avif&q=75`
+      )}
+      /${portraitSource.basename}.avif`
     )
     expect(parsedSrcSet[0].descriptor).toEqual(`75w`)
     expect(parsedSrcSet[1].src).toEqual(
-      `/_gatsby/image/${Buffer.from(portraitSource.url).toString(
-        `base64`
-      )}/${Buffer.from(`w=150&h=241&fm=avif&q=75`).toString(`base64`)}/${
-        portraitSource.basename
-      }.avif`
+      `/_gatsby/image/${base64URLEncode(portraitSource.url)}/${base64URLEncode(
+        `w=150&h=241&fm=avif&q=75`
+      )}/${portraitSource.basename}.avif`
     )
     expect(parsedSrcSet[1].descriptor).toEqual(`150w`)
     expect(parsedSrcSet[2].src).toEqual(
-      `/_gatsby/image/${Buffer.from(portraitSource.url).toString(
-        `base64`
-      )}/${Buffer.from(`w=300&h=481&fm=avif&q=75`).toString(`base64`)}/${
-        portraitSource.basename
-      }.avif`
+      `/_gatsby/image/${base64URLEncode(portraitSource.url)}/${base64URLEncode(
```

Comment:
Will `base64URLEncode` break our Cloud route? Can it be base64 decoded the same way?

Reply:
If so we might need to add some indication to the url that this is a new version of it 🤔

Reply:
No breaking changes with the cloud route! Using `Buffer.from(<string>, 'base64').toString()` will automatically handle both `base64` and `base64url` correctly. 

From the node docs on base64 decoding: https://nodejs.org/api/buffer.html#buffers-and-character-encodings

> 'base64url': base64url encoding as specified in RFC 4648, Section 5. When creating a Buffer from a string, this encoding will also correctly accept regular base64-encoded strings. When encoding a Buffer to a string, this encoding will omit padding.

Reply:
Isn't that saying it works the other way around? When decoding base64url encoded strings, regular base64 strings will also decode. So we should change our Cloud route to decode as base64url before we ship this

Reply:
Ope, check out this one! Our current router code should handle this properly


> 'base64': [Base64](https://en.wikipedia.org/wiki/Base64) encoding. When creating a Buffer from a string, this encoding will also correctly accept "URL and Filename Safe Alphabet" as specified in [RFC 4648, Section 5](https://tools.ietf.org/html/rfc4648#section-5). Whitespace characters such as spaces, tabs, and new lines contained within the base64-encoded string are ignored.

Reply:
Ah, sweet! Thanks 😄

Reply:
Totally!